package.json

```diff
@@ -82,5 +82,6 @@
     "stylelint-webpack-plugin": "^3.3.0",
     "tailwindcss": "^3.4.3",
     "typescript": "^5.4.2"
-  }
+  },
+  "packageManager": "pnpm@9.3.0+sha512.ee7b93e0c2bd11409c6424f92b866f31d3ea1bef5fbe47d3c7500cdc3c9668833d2e55681ad66df5b640c61fa9dc25d546efa54d76d7f8bf54b13614ac293631"
```

Comment:
Should this be added?

Reply:
@joaopcm good catch – can probably leave that out, it was autogenerated when I ran `pnpm i` 😅

Reply:
yeah, I hate when my CLI/IDE adds this automatically as well, lol

great! thanks!